features/step_definitions/user_steps.rb

```diff
@@ -233,6 +233,20 @@
   visit change_username_user_path(user)
 end
 
+When "I visit the change email page for {string}" do |login|
+  user = User.find_by(login: login)
+  visit change_email_user_path(user)
+end
+
+When "{string} old email {string} should be emailed" do |user, old_email|
```

Comment:
Rubocop already pointed it out, the username is unused here. I would propose to remove the username from the step name (something like "the email address {string} should be emailed") and to move this to `email_custom_steps.rb`